### PR TITLE
Add `.gitignore`

### PR DESCRIPTION
**Pull request by ArthurSonzogni**
Bug:https://github.com/ArthurSonzogni/FTXUI/issues/669

### CODE REVIEW

**Comment by ArthurSonzogni** [comment]
Hello @avighnac,
Could you please take a look?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+!CMakeLists.txt
+!LICENSE
+!README.md
+!codecov.yml
+!flake.lock
+!flake.nix
+!ftxui.pc.in
+!iwyu.imp
+
+# .gitbug directory:
```
Comment (avighnac) [inline .gitignore:18]: This looks like a typo.

**Review by avighnac — APPROVED** [review]
Other than that one typo, LGTM!

File: .gitignore (hunk 1 above)
Comment (ArthurSonzogni) [reply .gitignore]: Nice catch! Thanks!

**Pull request merged**